lib/derivative_rodeo/generators/base_generator.rb

```diff
@@ -106,7 +106,7 @@ def generated_files
           @generated_files << if generated_file.exist?
                                 generated_file
                               else
-                                log_message = "#{self.class}#generated_files :: " \
+                                log_message = "🤠🐮 #{self.class}#generated_files :: " \
```

Comment:
these break in k8s logging tools. I know they are fun, but can we skip them for the sake of consistency?

Reply:
**sigh** I'll remove them.  They aren't just fun, they are helpful to quick scan.